uxarray/grid/grid.py

```diff
@@ -1506,6 +1506,18 @@ def global_sphere_coverage(self):
         (i.e. contains no holes)"""
         return not self.partial_sphere_coverage
 
+    @property
+    def inverse_face_indices(self):
+        if self._is_subset:
+            return self._ds["inverse_face_indices"]
+
+    @property
+    def _is_subset(self):
+        """Boolean indicator for whether the Grid is from a subset or not."""
+        if "_is_subset" not in self._ds:
+            self._ds["_is_subset"] = False
+        return self._ds["_is_subset"]
```

Comment:
A simple `self._is_subset` should suffice instead of storing it inside of the internal data.

I would suggest adding a `self._is_subset` attribute inside of the constructor, which is set to false by default. 

https://github.com/UXARRAY/uxarray/blob/7c4f3e10019229816525f7796545d191f65c42d4/uxarray/grid/grid.py#L158-L163